doc/changes/0.10.4.rst

```diff
@@ -5,47 +5,49 @@
 0.10.4.dev
 ==========
 
+**Released April 2024**
+
+HIGHLIGHTS
+----------
+
+- :bdg-dark:`Code` Use the appropriate HTML templates for :class:`nilearn.maskers.MultiNiftiMapsMasker` and :class:`nilearn.maskers.MultiNiftiLabelsMasker` reports that would lead to empty reports for ``MultiNiftiMapsMasker`` (:gh:`4310` by `Rémi Gau`_).
+
 Fixes
 -----
 
 - :bdg-dark:`Code` Fix plotting of carpet plot due to a change in the coming version of matplolib (3.9.0) (:gh:`4279` by `Rémi Gau`_).
 - :bdg-dark:`Code` Fix errant warning when using ``stat_type`` in :func:`nilearn.glm.compute_contrast` (:gh:`4257` by `Eric Larson`_).
 - :bdg-dark:`Code` Fix when thresholding is applied to images by GLM reports (:gh:`4258` by `Rémi Gau`_).
```

Comment:
I think that we have an extra line between each item of the changelog. Can you check the previous version release? Also this is an implicit thing we should probably mention it somewhere.

Reply:
Yes indeed! I will add that to the same PR for directives.